doc/conf.py

```diff
@@ -137,8 +139,5 @@
 # -- Intersphinx setup ----------------------------------------------------
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {
-    "python": ("https://docs.python.org/3/", None),
-    "numpy": ("https://numpy.org/devdocs/", None),
-    "sklearn": ("https://scikit-learn.org/stable/", None),
-}
+
+intersphinx_mapping = get_intersphinx_mapping(["python", "numpy", "sklearn"])
```

Comment:
There was a backward incompatible change made here :)
```suggestion
intersphinx_mapping = get_intersphinx_mapping(packages=["python", "numpy", "sklearn"])
```